include/itkMaxPhaseCorrelationOptimizer.h

```diff
@@ -37,10 +37,6 @@ namespace itk
  *  If interpolation method is None, the shift is estimated with pixel-level
  *  precision. Otherwise the requested interpolation method is used.
  *
- * \author Jakub Bican, jakub.bican@matfyz.cz, Department of Image Processing,
- *         Institute of Information Theory and Automation,
- *         Academy of Sciences of the Czech Republic.
- *
```

Comment:
We should retain these author comments whenever possible.

Reply:
OK, I will remove this commit.